CodePush.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-var { Alert } = require("./AlertAdapter");
-var NativeCodePush = require("react-native").NativeModules.CodePush;
-var PackageMixins = require("./package-mixins")(NativeCodePush);
-var requestFetchAdapter = require("./request-fetch-adapter.js");
-var Sdk = require("code-push/script/acquisition-sdk").AcquisitionManager;
-var semver = require("semver");
+import { Alert } from "./AlertAdapter";
+let NativeCodePush = require("react-native").NativeModules.CodePush;
```

Comment:
Are you sure that works? https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=true&spec=false&code=import%20%7B%20NativeModules%20%3A%20%7B%20CodePush%20as%20NativeCodePush%20%7D%20%7D%20from%20%22react-native%22%3B